src/workloads/execution/SSBColumnStoreIndex.yml

```diff
@@ -0,0 +1,791 @@
+      PhaseConfig:
+        Repeat: *TestRepeatCount
+        Database: *db
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: Q2_1Query
+          OperationName: aggregate
+          OperationCommand: *Q2_1Query
+
+- Name: Q2_2
+  Type: CrudActor
+  Phases:
+    OnlyActiveInPhases:
+      Active: [11]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *TestRepeatCount
+        Database: *db
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: Q2_2Query
+          OperationName: aggregate
+          OperationCommand: *Q2_2Query
+
+- Name: Q2_3
+  Type: CrudActor
+  Phases:
+    OnlyActiveInPhases:
+      Active: [13]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *TestRepeatCount
+        Database: *db
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: Q2_3Query
+          OperationName: aggregate
+          OperationCommand: *Q2_3Query
+
+- Name: Q3
+  Type: CrudActor
+  Phases:
+    OnlyActiveInPhases:
+      Active: [15]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *TestRepeatCount
+        Database: *db
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: Q3Query
+          OperationName: aggregate
+          OperationCommand: *Q3Query
+
+- Name: Q4_1
+  Type: CrudActor
+  Phases:
+    OnlyActiveInPhases:
+      Active: [17]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *TestRepeatCount
+        Database: *db
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: Q4_1Query
+          OperationName: aggregate
+          OperationCommand: *Q4_1Query
+
+- Name: Q4_2
+  Type: CrudActor
+  Phases:
+    OnlyActiveInPhases:
+      Active: [19]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *TestRepeatCount
+        Database: *db
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: Q4_2Query
+          OperationName: aggregate
+          OperationCommand: *Q4_2Query
+
+- Name: Q4_3
+  Type: CrudActor
+  Phases:
+    OnlyActiveInPhases:
+      Active: [21]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *TestRepeatCount
+        Database: *db
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: Q4_3Query
+          OperationName: aggregate
+          OperationCommand: *Q4_3Query
+
+# The 'DropColumnstoreIndexes' actor drops columnstore indexes.
+- Name: DropColumnstoreIndex
+  Type: RunCommand
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [22]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: 1
+        Database: *db
+        Operations:
+        - OperationName: RunCommand
+          OperationCommand:
+            dropIndexes: *Collection
+            index: *ColumnStoreIndexName
```

Comment:
[suggestion] Feel free to ignore, but if it were me, I would consider putting the DropColumnstoreIndex in phase 23 and changing the Quieces to all be in even phases. Then document that odd phases are setup and even phases are test related (except for the drop)  in the  the workload header. 

Operationally this shouldn't affect anything, it's only an extra fsync between CSI Q4_3 and the drop index.

Reply:
Makes sense, fixed in 88de40e2b8e3ed8b2d3a82106f435cd65d00ba3a